### ISSUE TITLE
Question: Continuous Delivery (CD) from crates.io ?

### ISSUE THREAD

**Issue by biancode** [rust-lang/crates.io#1554]
Dear crates.io,

I asked via E-Mail and they pleased me to post here.

The question is, wouldn't it be nice to have a default binary download for all on crates.io like https://mvnrepository.com/ ?
It could just provide Linux, macOS, Windows, but I think it could be more.

idea: 

* provide a select on page for all mainly used build systems (linux, macOS, windows ...) 
* a subsystem select (fedora, debian, raspian, x86_64  ...) 
* and a download button for the binaries zip/tar.gz 
** if the user click the button
*** then the system could check for previous builds and send the binaries aszip/tar.gz or it could build them once and send it to the user

With that you have just the requested bins compressed on the server and it has to build only once for all.

I mean that will support users and developers to get a much better experience on using rust by crates.

That would be a great step forward to provide rust tools like command line tools to the community, wouldn't it?
It could centralize rust deployments to crates.io and give a good experience to new rust users - one place to get the latest release.

Happy coding!
Klaus

**Comment by carols10cents** [comment]
Hi, thanks for the suggestion! Unfortunately, this is not a responsibility we'd like crates.io to have at this time for the reasons stated in [duplicate issue #1396](https://github.com/rust-lang/crates.io/issues/1396#issuecomment-389698099).
